examples/policyGenerator-kustomize.yaml

```diff
@@ -13,3 +13,8 @@ policies:
 - name: myapp
   manifests:
     - path: input-kustomize/
+      patches:
+      - metadata:
+          name: "myname"
+      openapi:
+        path: schema.json
```

Comment:
The examples here are meant to be generated using `make generate`, but the schema isn't available. Options in this case are to either remove or comment out this addition (the `examples/` directory isn't necessarily comprehensive) or add a `schema.json` file so that `make generate` succeeds.

Reply:
Fixed. I removed it, actually this was not a good example as it was not pointing to a manifest.